packages/gafl-webapp-service/src/pages/multibuy/remove-licence/__tests__/update-transaction.spec.js

```diff
@@ -1,35 +1,33 @@
 import { ADD_LICENCE, REMOVE_LICENCE } from '../../../../uri.js'
 import updateTransaction from '../update-transaction.js'
 
-const createCache = (cache = {}) => ({
-  helpers: {
-    status: {
-      setCurrentPermission: () => cache.status?.permissions[0] || {},
-      get: () => cache.status || { permissions: [getStatusPermission(), getStatusPermission(), getStatusPermission()] },
-      set: () => cache.status || (() => {})
-    },
-    transaction: {
-      getCurrentPermission: () => cache.transaction?.permissions[0] || {},
-      get: () =>
-        cache.transaction || {
-          permissions: [getTransactionPermissionOne(), getTransactionPermissionTwo(), getTransactionPermissionThree()]
-        },
-      set: () => cache.transaction || (() => {})
-    },
-    page: {
-      get: () => cache.page || { permissions: [getPagePermission(), getPagePermission(), getPagePermission()] },
-      set: () => cache.page || (() => {})
-    },
-    addressLookup: {
-      getCurrentPermission: () => cache.addressLookup?.permissions[0] || {},
-      get: () => cache.addressLookup || { permissions: [{}, {}, {}] },
-      set: () => cache.addressLookup || (() => {})
+const getMockRequest = (opts = {}, statusSet = () => {}, statusSetCurrentPermission = () => {}) => ({
+  cache: () => ({
+    helpers: {
+      status: {
+        setCurrentPermission: statusSetCurrentPermission,
+        get: () => opts.cache.status || { permissions: [getStatusPermission(), getStatusPermission(), getStatusPermission()] },
+        set: statusSet
+      },
+      transaction: {
+        getCurrentPermission: () => opts.cache.transaction?.permissions[0] || getTransactionPermissionThree(),
+        get: () =>
+          opts.cache.transaction || {
+            permissions: [getTransactionPermissionOne(), getTransactionPermissionTwo(), getTransactionPermissionThree()]
+          },
+        set: () => opts.cache.transaction || (() => {})
+      },
+      page: {
+        get: () => opts.cache.page || { permissions: [getPagePermission(), getPagePermission(), getPagePermission()] },
+        set: () => opts.cache.page || (() => {})
+      },
+      addressLookup: {
+        getCurrentPermission: () => opts.cache.addressLookup?.permissions[0] || {},
+        get: () => opts.cache.addressLookup || { permissions: [{}, {}, {}] },
```

Comment:
Again, like transaction cache, you never actually vary this, so this could be the hardcoded value